api/src/certification/evaluation/domain/models/Candidate.js

```diff
@@ -0,0 +1,5 @@
+export class Candidate {
+  constructor({ accessibilityAdjustmentNeeded } = {}) {
+    this.accessibilityAdjustmentNeeded = accessibilityAdjustmentNeeded;
+  }
+}
```

Comment:
Notre model etant le coeur de notre travail, je propose d'y ajouter un peu de doc (qui permettra a nos IDE de faire de l'inference, de la detection de problemes et autocompletion) + d'empecher des detournements via un casting explicite a l'assignement

```suggestion
export class Candidate {
   /**
   * @param {Object} params
   * @param {boolean} [params.accessibilityAdjustmentNeeded]
   */
  constructor({ accessibilityAdjustmentNeeded } = {}) {
    this.accessibilityAdjustmentNeeded = !!accessibilityAdjustmentNeeded;
  }
}
```